FileBaseContext/Storage/FileBaseContextTable.cs

```diff
@@ -31,7 +31,7 @@ public FileBaseContextTable(IEntityType entityType, IServiceProvider serviceProv
         _options = options;
```

Comment:
nit: I can't comment on line 30 since it's not in the diff, but should that `GetService()` also be `GetRequiredService()`?

Reply:
fixed in 1a5c0c51b15ac1c2e0fd7f85123225f2f5956835